### DISCUSSION TITLE
Self-hosting

### DISCUSSION THREAD

**Discussion by jgoestl** [BernhardSchlegel/BierBot-Bricks#18] [Q&A]
Hi, is it possible to self host the service running at bricks.bierbot.com?

**Comment by BernhardSchlegel** [comment]
Hi Julian,

unfortunately this is currently not possible. However, if you are interested in a collaboration [please reach out](https://bierbot.com/contact/) to us!

Cheers,
Bernhard.

**Reply by jgoestl** [reply]
Is there something planned? I love it to self host all my smarthome things and so the smart brewing things also. :-)

**Reply by BernhardSchlegel** [reply]
I do absolutely understand that ! However, I can safely exclude that the backend will be open sourced short-term (next 12 months). Sorry!
